voc_builder/config.py

```diff
@@ -1,9 +1,11 @@
 """Store configuration for project"""
+import os
 from pathlib import Path
 
 # Testings should patch these variables
 #
 # The default path for storing the vocabulary book
-DEFAULT_CSV_FILE_PATH = Path('~/aivoc_builder.csv').expanduser()
+aivoc_home = os.environ.get("AIVOC_HOME", "~")
```

Comment:
Please rename the name of this config entry to `AIVOC_DATA_DIR`. Reason: There will be more config entries in the furture, `DATA_DIR` more precise than `HOME`.